public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>o2znz</title>
+</head>
+<body>
+  <div id="app"></div>
```

Comment:
note: for combining our modules, we'll probably need to change this to something less generic / specific to each module